### PR TITLE
Adds intro text for individual form controls

### PR DESCRIPTION
**Pull request by carodew**
Does not include range slider intro text. 

Current page template only allows for one large intro text for the entire page, but we have intro content for each component. I just added the p tags in with the appropriate class.

### CODE REVIEW

File: _elements/form-controls.md [hunk 1]
```diff
@@ -286,7 +290,7 @@ lead: Intro text on what is included in this section and how to use it. No more
       </div>
     </div>
   </fieldset>
-
+c
```
Comment (msecret) [inline _elements/form-controls.md]: Is this supposed to be here?

File: _elements/form-controls.md [hunk 2]
```diff
@@ -327,6 +331,7 @@ lead: Intro text on what is included in this section and how to use it. No more
 </div>
 
 <h2 class="usa-heading" id="range-slider">Range slider</h2>
+<!--<p class="usa-font-lead">placeholder for range slider intro text </p>-->
```
Comment (msecret) [inline _elements/form-controls.md]: You don't have to make this a comment, you can just remove it.

File: _elements/form-controls.md (hunk 1 above)
Comment (carodew) [inline _elements/form-controls.md]: fixed in e4fb0b8

File: _elements/form-controls.md (hunk 2 above)
Comment (carodew) [inline _elements/form-controls.md]: removed in 48886ee

**Pull request merged**